lib/iterator.js

```diff
@@ -19,6 +19,7 @@ function Iterator (db, prefix, opts) {
   this._start = 0
   this._updated = false
   this._gt = !!opts.gt
+  this._sortOrder = opts.latest ? 'latest' : 'hash'
```

Comment:
Use a number instead of 'latest'/'hash' (will be much faster and this is very hot code)

Reply:
Or simply ._sortLatest :)

Reply:
👍 will change +1